lib/matplotlib/collections.py

```diff
@@ -642,12 +643,14 @@ def set_color(self, c):
         """
         Set both the edgecolor and the facecolor.
 
-        ACCEPTS: matplotlib color arg or sequence of rgba tuples
-
         .. seealso::
 
             :meth:`set_facecolor`, :meth:`set_edgecolor`
                For setting the edge or face color individually.
+
+        Parameters
+        ----------
+        c : matplotlib color arg or sequence of rgba tuples
```

Comment:
Is there a way to link the color arg page here?

Reply:
Probably, but I'd rather you just open an issue and we try to resolve that as a separate thing.